manage_proj_edit_page.php

```diff
@@ -559,7 +559,7 @@
 		<fieldset>
 			<?php echo form_security_field( 'manage_proj_ver_add' ) ?>
 			<input type="hidden" name="project_id" value="<?php echo $f_project_id ?>" />
-			<input type="text" class="input-sm" name="version" size="32" maxlength="64" />
+			<input type="text" class="input-sm" required name="version" size="32" maxlength="64" />
```

Comment:
While you are on it, it would be good to change also the category field some lines above.

Not sure if we should add
`<span class="required">*</span>`
like we have it for the project name and the user name on the same page.

Reply:
> While you are on it, it would be good to change also the category field some lines above.

Right, I meant to add the required property here too initially, then I forgot... Will do.
Note that no change is necessary in the "add" script, as the empty string case was already dealt with.

> Not sure if we should add
> `<span class="required">*</span>`
> like we have it for the project name and the user name on the same page.

I would vote against that - it's not a "regular" form, and I believe it should be quite obvious that the field is mandatory, without this additional visual hint, which looks a bit awkward IMHO

![image](https://user-images.githubusercontent.com/449891/109614903-859adf00-7b33-11eb-88d8-e7f24e1612de.png)